hudi-spark-datasource/hudi-spark/src/main/java/org/apache/spark/sql/hudi/command/procedures/ProcedureConstEnum.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.sql.hudi.command.procedures;
+
+import java.util.Arrays;
+
+public enum ProcedureConstEnum {
+  TABLE("table"),
+  PATH("path"),
+  LIMIT("limit"),
+  TIMESTAMP("timestamp"),
+  ACTION("action"),
+  SIZE("size"),
+  OP("op"),
+  INSTANT("instant"),
+  INSTANT_TIME("instant_time"),
+  COMMIT_TIME("commit_time"),
+  USER("user"),
+  COMMENTS("comments"),
+  GROUPS("groups"),
+  PREDICATE("predicate"),
+  ORDER("order"),
+  PARTITION("partition"),
+  SAVEPOINT_TIME("savepoint_time"),
+  CREATE_SAVEPOINT_RESULT("create_savepoint_result"),
+  ROLLBACK_SAVEPOINT_RESULT("rollback_savepoint_result"),
```

Comment:
I don't think input and output parameters should be put into constant enums. This will be a lot.